test/acceptance/README.md

```diff
@@ -1,10 +1,230 @@
+
+For that there is `TEST_ACCEPTANCE_START_SBO` environment variable that needs to be set to `remote`.
+
+The remote cluster where the SBO should be running is determined by the `KUBECONFIG`.
+
+To run acceptance tests with remote instance of SBO:
+
+```bash
+TEST_ACCEPTANCE_START_SBO=remote make test-acceptance
+```
+
+Compared with the defaults this command only:
+* Deletes the old namespace from any previous run,
+* Creates a new namespace with generated name like `test-namespace-xxxxxxxx`,
+* Executes the acceptance tests.
+
+The `remote` option requires to specify the namespace where the operator is running by setting the `SBO_NAMESPACE` env variable.
+
+To run acceptance tests with remote instance of SBO running in a given namespace (e.g. `sbo`):
+
+```bash
+TEST_ACCEPTANCE_START_SBO=remote SBO_NAMESPACE=sbo make test-acceptance
+```
+
+
+### Run a sub-set of features or scenarios
+
+It is possible to run a sub-set of features or even a single scenario (for example, when you are working on a new scenario and you need to run it over and over). The way how to do it is by "marking" that feature or scenario by a "tag" - directly in the particular `*.feature` file - in a form of `@<tag>` (e.g. `@wip`). Something like the following:
+
+```gherkin
+Feature: ...
+
+   @wip
+   Scenario: Bind service 1 to app 1
+   ...
+
+   Scenario: Bind service 1 to app 2
+   ...
+```
+
+Now, when you want to execute just that scenario out of the whole suite, use the `TEST_ACCEPTANCE_TAGS` environmental variable to set the tag.
+
+To run single scenario tagged with `@wip`:
+
+```bash
+TEST_ACCEPTANCE_TAGS=@wip make test-acceptance
+.
+.
+.
+
+```
+
+Only the scenario or feature tagged with the particular tag (`@wip` in case of our example) will be executed, and all of the others will be simply skipped.
+
+```bash
+.
+.
+.
+
+1 feature passed, 0 failed, 3 skipped
+1 scenario passed, 0 failed, 16 skipped
+10 steps passed, 0 failed, 179 skipped, 0 undefined
+Took 0m49.231s
+```
+
+## Inspect test results
+
+During the test execution the result of individual steps are shown as the execution goes. That is the most obvious way to see the results, but it is far from elegant - especially with a growing number of scenarios. However, the framework also records the test results in a standardized way of xUnit format, which can be found under `$WORKSPACE/out/acceptance-tests` directory.
+
+As mentioned above those `*.xml` test results files are in xUnit format, which is standardized although it is not human readable. To make it readable a tool for visualizating is needed to help us.
+
+There's one tool in particular that is meant for BDD-like results and that is [Allure](https://docs.qameta.io/allure/)
+
+### Install Allure tool
+
+Follow the [instructions](https://docs.qameta.io/allure/#_installing_a_commandline).
+
+### Show test result report
+
+```bash
+allure serve out/acceptance-tests
+```
+
```

Comment:
See above the idea bout using a Docker image...

Reply:
I've introduced a docker image for generating the allure reports...